docs/platform/sessions/batch.md

```diff
@@ -39,11 +69,20 @@ Execute containers programmatically using the `canfar` command-line client:
 # Ensure you are logged in first
 canfar auth login
 
-# Submit a job
+# Submit a flexible session job (default - auto-scaling resources)
```

Comment:
These need to be modified @WenbinWL 

Flexible jobs just omit cpu/ram/gpu and there is no command or image option, this should look like,

```
canfar launch -n data-reduction headless skaha/astroml:latest -- python /arc/path/to/file.py
```